silver/models/payment_processors/generics.py

```diff
@@ -0,0 +1,105 @@
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+class PaymentProcessorMeta(type):
+    def __repr__(self):
+        return self.name
+
+    def __unicode__(self):
+        return unicode(self.name)
+
+    def __str__(self):
+        return str(self.name)
+
+
+class PaymentProcessorTypes(object):
+    Manual = "manual"
+    Automatic = "automatic"
+    Triggered = "triggered"
+    Mixed = "mixed"
```

Comment:
Yeah, you're right, I'll remove that one. I'll update the wiki with a proper explanation soon.